metricbeat/module/system/cpu/_meta/fields.yml

```diff
@@ -6,14 +6,14 @@
     - name: cores
       type: long
       description: >
-        The number of CPU cores.
+        The number of CPU cores. The CPU percentages can range from `[0, 100% * cores]`.
```

Comment:
Did you intend to add this here? Is it to explain how is the number of cores useful?